packages/polaris-typeorm/src/utils/query-builder-util.ts

```diff
@@ -0,0 +1,46 @@
+import { EntityFilter } from '@enigmatis/polaris-common';
+import { SelectQueryBuilder } from 'typeorm';
+
+export const addDateRangeCriteria = (
+    qb: SelectQueryBuilder<any>,
+    dateRangeFilter: EntityFilter,
+    entityTableName: string,
+) => {
+    if (dateRangeFilter.creationTimeFilter?.gt) {
+        qb.andWhere(`${entityTableName}.creationTime > :gt`, {
+            gt: dateRangeFilter.creationTimeFilter?.gt,
+        });
+    } else if (dateRangeFilter.creationTimeFilter?.gte) {
```

Comment:
i would have switched gte to be the if, cause if someone put both gte and gt, i would have wanted gte to work